macros/src/auto_args.rs

```diff
@@ -254,7 +255,7 @@ macro_rules! build_rpc_trait {
 /// A wrapper type without an implementation of `Deserialize`
 /// which allows a special implementation of `Wrap` for functions
 /// that take a trailing default parameter.
-pub struct Trailing<T: Default + Deserialize>(pub T);
+pub struct Trailing<T: DeserializeOwned>(pub T);
```

Comment:
fixed. If we are about to remove all bounds, I would prefer to do it in a separate pr ;)